app/views/content/blog/product-engineer-turned-science-teacher.md

```diff
@@ -0,0 +1,61 @@
+keywords:
+  - teacher training advisers
+  - becoming a teacher
+  - applications
+  - teacher training
+  - career changer
+  - science teacher
+tags:
+  - becoming a teacher
+  - career change to teaching
+---
+
+## What inspired you to become a teacher?
+
+I was very good at maths at school but loved physics, so a career in engineering seemed the obvious choice.
+
+I thoroughly enjoyed my first role in industry as a production engineer, but a promotion took me away from the practical engineering and I no longer enjoyed what I was doing.
+
+I tried many things until I realised that working with young people and helping them develop skills truly inspired me. So I applied for a PGCE at a University.
+
+As soon as I set foot in the classroom, I knew it was for me. There were nerves and I didn’t enjoy presenting in assemblies to start with. But, in the classroom with the students, I enjoyed the challenge.
+
+## Why do you think it's important that engineering and technology are taught to children?
+
+The future of the country, and the planet, is becoming more and more dependent on highly STEM educated individuals who can identify the problems we face and hopefully develop solutions.
+
+As educators, we have a duty to help students achieve this. They'll understand the importance of the role of engineering and technology in the future of their community, country, and the planet.
+
+Highlighting that every industry is heavily dependent on engineering will inspire them to study hard in the relevant areas of the curriculum, allowing them to become future leaders in engineering and technology.
+
+## Would you encourage others to change careers to teaching? 
+
+I would thoroughly recommend teaching to other engineers. The intellectual challenge is very high, not so much in understanding the content, but how to impart your knowledge to others who may not be as keen to learn as previous colleagues.
+
+Classroom management is a real opportunity to utilise creative problem-solving methods that have an immediate impact and can change the lives of young people.
+
+## How is having an engineering background beneficial to teaching children? 
+
+Children enjoy challenges and solving problems. Coming from an engineering background where the day-to-day job is exactly that means we can be on the same wavelength as the students.
+
+I have a wealth of real-world examples to fall back on when teaching about maths or physics. I know its relevance, how interesting and challenging it can be and the euphoria of successfully cracking a difficult problem or managing a team that completes a major project. 
+
+Having enough experience to answer the perennial question, “Why do we need to learn this?” is invaluable.
+
+## It may be daunting for someone to switch to a teaching role - are skills easily transferable from engineer/technician to teacher? 
```

Comment:
Hi @EllieNodder it's the dash question again. In preview it looks like a hyphen in this heading.

Reply:
have changed them and it seems to be working for me now - is it coming up right for you?